pycsob/utils.py

```diff
@@ -16,6 +17,10 @@
     from datetime import datetime
 
 
+class CsobCommunicationError(Exception):
```

Comment:
Can exists some parent of all CSOB exceptions?

Reply:
Currently, there's no other CSOB exception in the code.  Do you want me to make one general CSOB exception?

Reply:
Moved to a separate file and inherited from baseCSOB Exception.